Bottom: e9ffc684f70d1dead731a7736023ec6718e13ee3
Top:    baeaa6770a2ee835729fd4c140486ef50f183ede
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:07 +0000

Disable debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index ab316e7..7039457 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1252,7 +1252,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                                 copper_appearance,
                                 NULL));
 
-      break;
+      //break;
     }
 
 
@@ -1264,10 +1264,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
